.dockerignore

```diff
@@ -0,0 +1,2 @@
+*
+!_local/scripts
```

Comment:
We can remove the `.dockerignore` and instead be explicit on the files that need to be copied into container.

Reply:
Hi @pjbgf so without the `.dockerignore` all of the files are copied to the build "context" when the build happens, so it slows down the build. Do you still want to remove the file?